node/src/components/storage/tests.rs

```diff
@@ -1854,33 +1925,78 @@ fn should_restrict_returned_blocks() {
     let mut harness = ComponentHarness::default();
     let verifiable_chunked_hash_activation = EraId::new(u64::MAX);
 
-    let mut storage = storage_fixture(&harness, verifiable_chunked_hash_activation);
+    let storage = storage_fixture(&harness, verifiable_chunked_hash_activation);
 
     // Create the following disjoint sequences: 1-2 4-5
     [1, 2, 4, 5].iter().for_each(|height| {
         let (block, _) = random_block_at_height(&mut harness.rng, *height, Block::random_v1);
-        storage.write_block(&block).unwrap();
+        storage.storage.write_block(&block).unwrap();
     });
     // The available range is 4-5.
```

Comment:
I know this is not related to this PR, but I believe this comment should now say:
```
The lowest available block is 4.
```